daft/expressions/expressions.py

```diff
@@ -275,10 +275,15 @@ def __rand__(self, other: Expression) -> Expression:
         return Expression._from_pyexpr(expr._expr & self._expr)
 
     def __or__(self, other: Expression) -> Expression:
-        """Takes the logical OR of two boolean expressions (``e1 | e2``)"""
+        """Takes the logical OR of two boolean or integer expressions (``e1 | e2``)"""
         expr = Expression._to_expression(other)
         return Expression._from_pyexpr(self._expr | expr._expr)
 
+    def __xor__(self, other: Expression) -> Expression:
+        """Takes the logical XOR of two boolean or integer expressions (``e1 ^ e2``)"""
+        expr = Expression._to_expression(other)
+        return Expression._from_pyexpr(self._expr ^ expr._expr)
+
```

Comment:
good catch!

Reply:
I'm curious why pyspark doesn't have this though, even though they have all the other ones. Maybe someone has a good reason for that, just something to check again

Reply:
🤷 maybe ppl just don't use it that much